cask-repair

```diff
@@ -561,12 +561,23 @@ for cask in "${@}"; do
 
   git commit "${cask_file}" --message "${commit_message}" --quiet
   git push --force "${cask_repair_remote_name}" "${cask_branch}" --quiet 2> "${submission_error_log}"
+  gitpush_result="${?}"
 
   # Fix common push errors
-  if [[ "${?}" -ne 0 ]] && grep --quiet 'shallow update not allowed' "${submission_error_log}"; then
+  if [[ "${gitpush_result}" -ne 0 ]] && grep --quiet 'shallow update not allowed' "${submission_error_log}"; then
     echo 'Push failed due to shallow repo. Unshallowing…'
     HOMEBREW_NO_AUTO_UPDATE=1 brew tap --full "caskroom/$(current_tap)"
-    git push --force "${cask_repair_remote_name}" "${cask_branch}" --quiet
+    git push --force "${cask_repair_remote_name}" "${cask_branch}" --quiet 2> "${submission_error_log}"
+    gitpush_result = "${?}"
+  fi
+
+  if [[ "${gitpush_result}" -ne 0 ]] && grep --quiet 'error:' "${submission_error_log}"; then
+    warning_message 'There was an error submitting the pull request.'
```

Comment:
No repeated `warning_message`. Make it a single one instead: `warning_message 'There were errors while pushing:`.